pkgs/development/interpreters/python/cpython/2.7/default.nix

```diff
@@ -335,8 +335,14 @@ in with passthru; stdenv.mkDerivation ({
       license = lib.licenses.psfl;
       platforms = lib.platforms.all;
       maintainers = with lib.maintainers; [ fridh ];
-      # Higher priority than Python 3.x so that `/bin/python` points to `/bin/python2`
-      # in case both 2 and 3 are installed.
-      priority = -100;
```

Comment:
Even with the recent patches for Python 2.7, we should probably go and give higher priority to Python 3 right now and see if something breaks.